.github/workflows/build.yml

```
@@ -55,7 +55,17 @@ jobs:
           run: dotnet build SentryNoSamples.slnf -c Release /p:CopyLocalLockFileAssemblies=true
 
         - name: Test
-          run: dotnet test SentryNoSamples.slnf -c Release --no-build -l "GitHubActions;report-warnings=false"
+          run: dotnet test SentryNoSamples.slnf -c Release --no-build -l "GitHubActions;report-warnings=false" -l "trx;LogFileName=${{ matrix.os }}-test-results.trx"
+
+        - name: Create Test Report
+          uses: phoenix-actions/test-reporting@v6
```

This isn't a verified action (https://github.com/marketplace/actions/test-reporting) so please pin to a commit SHA after verifying its content:

https://github.com/phoenix-actions/test-reporting/releases/tag/v6
->
https://github.com/phoenix-actions/test-reporting/commit/4b942144e31c4039ac808d9b29945864b628a574

Browse through: https://github.com/phoenix-actions/test-reporting/tree/4b942144e31c4039ac808d9b29945864b628a574


```suggestion
          uses: phoenix-actions/test-reporting@4b942144e31c4039ac808d9b29945864b628a574
```